app/controllers/client_api/notifications_controller.rb

```diff
@@ -1,23 +1,28 @@
 module ClientApi
   class NotificationsController < ApplicationController
-    before_action :last_notifications, only: :recent_notifications
-
     def recent_notifications
       respond_to do |format|
         format.json do
           render template: '/client_api/notifications/index',
                  status: :ok,
-                 locals: { notifications: @recent_notifications }
+                 locals: {
+                   notifications:
+                    UserNotification.recent_notifications(current_user)
+                 }
         end
       end
+      # clean the unseen notifications
+      UserNotification.seen_by_user(current_user)
     end
 
-    private
-
-    def last_notifications
-      @recent_notifications =
-        UserNotification.recent_notifications(current_user)
-      UserNotification.seen_by_user(current_user)
+    def unreaded_notifications_number
```

Comment:
If I can be a Grammar nazi 🙂, past tense of `read` is also `read`, so please change this to `unread_notifications_number`, or even better, `unread_notifications_count` sounds better to me 🥇.